sdk/keyvault/microsoft-azure-keyvault/pom.xml

```diff
@@ -56,7 +56,7 @@ the MIT License. See License.txt in the project root for license information. --
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-client-runtime</artifactId>
-      <version>1.6.15</version> <!-- {x-version-update;com.microsoft.azure:azure-client-runtime;external_dependency} -->
+      <version>1.7.2</version> <!-- {x-version-update;com.microsoft.azure:azure-client-runtime;external_dependency} -->
```

Comment:
Should this have the relocation section added since there is a Track 2 client for LeyVault?

Reply:
I didn't want to mix two different issue fixes. I can create another PR that addresses all the relocations.

Reply:
Does LeyVault fall along some ley lines?! 😆

Reply:
I am working on a PR for relocation section in Track1 KV POMs. Should be out today.

Reply:
I prefer a separate PR for the relocations too, thanks!